### PR TITLE
Cephfs

### PR DESCRIPTION
**Pull request by krigbaum**
Adds driver for ceph file systems

### CODE REVIEW

**Comment by gondor** [comment]
Thanks for contributing.  Before we review can you please run a ```make format``` or ```gofmt -s -w -l .``` and commit to this PR.

**Comment by krigbaum** [comment]
Sorry.  I'm still a bit fresh to idomatic go.  I ran the gofmt -s -w -l.

**Comment by krigbaum** [comment]
Closed by mistake.

**Comment by krigbaum** [comment]
Added format changes from gofmt -a -w -l.

File: netshare/drivers/nfs.go [hunk 1]
```diff
@@ -106,6 +107,12 @@ func (n nfsDriver) mountVolume(source, dest string, version int) error {
 	var cmd string
 
 	options := merge(n.mountm.GetOptions(dest), n.nfsopts)
+	for key, value := range options {
+		fmt.Printf("key = %d  -- Value = %s\n\r", key, value)
```
Comment (gondor) [inline netshare/drivers/nfs.go]: Is this left over debug?

File: netshare/drivers/nfs.go [hunk 2]
```diff
@@ -31,6 +31,7 @@ func NewNFSDriver(root string, version int, nfsopts string) nfsDriver {
 	}
 
 	if len(nfsopts) > 0 {
+		fmt.Printf("\t***** From NewNFSDriver: There ARE %d nfsopts\n\r", len(nfsopts))
```
Comment (gondor) [inline netshare/drivers/nfs.go]: Is this left over debug?

**Comment by gondor** [comment]
LGTM, two comments regarding left over debug/output.

Also could you send me docs to add to netshare.containx.io ?  See http://netshare.containx.io/docs/nfs for an example.  Feel free to take a stab at adding this yourself if it's easier, otherwise I don't mind making the update.  Repo if you want to take a look is at: https://github.com/ContainX/netshare.containx.io

**Comment by krigbaum** [comment]
I deleted the debug print statements.  Still working on the docs.

**Comment by gondor** [comment]
LGTM, thanks for the contribution!

**Pull request merged**